Samples/kotlin/src/main/res/layout/activity_routing.xml

```diff
@@ -16,9 +16,15 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <include
-        android:id="@+id/nav_overlay_container"
-        layout="@layout/navigation_overlay" />
+    <com.phunware.kotlin.sample.NavigationOverlayView
```

Comment:
No need to use an include for this, as there's only a single activity layout now.  We can just use the custom View directly in the layout.

Reply:
hope i dont get merge conflict, but shouldnt be too difficult to resolve

Reply:
Probably merge conflicts will happen, just need to replace the old include with this block.